src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -0,0 +1,226 @@
+import _ from 'underscore';
+import React from 'react';
+import {View, Image} from 'react-native';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import MenuItem from '../../components/MenuItem';
+import {Paycheck, Bank, Lock} from '../../components/Icon/Expensicons';
+import styles from '../../styles/styles';
+import TextLink from '../../components/TextLink';
+import Button from '../../components/Button';
+import Icon from '../../components/Icon';
+import colors from '../../styles/colors';
+import Navigation from '../../libs/Navigation/Navigation';
+import CONST from '../../CONST';
+import TextInputWithLabel from '../../components/TextInputWithLabel';
+import AddPlaidBankAccount from '../../components/AddPlaidBankAccount';
+import CheckboxWithLabel from '../../components/CheckboxWithLabel';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import exampleCheckImage from '../../../assets/images/example-check-image.png';
+import Text from '../../components/Text';
+import {setupWithdrawalAccount} from '../../libs/actions/BankAccounts';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+class BankAccountStep extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.toggleTerms = this.toggleTerms.bind(this);
+        this.addManualAccount = this.addManualAccount.bind(this);
+        this.addPlaidAccount = this.addPlaidAccount.bind(this);
+
+        this.state = {
+            // One of CONST.BANK_ACCOUNT.SETUP_TYPE
+            bankAccountAddMethod: undefined,
+            hasAcceptedTerms: true,
+            routingNumber: '',
+            accountNumber: '',
+        };
+    }
+
+    toggleTerms() {
+        this.setState(prevState => ({
+            hasAcceptedTerms: !prevState.hasAcceptedTerms,
+        }));
+    }
+
+    /**
+     * @returns {Boolean}
+     */
+    canSubmitManually() {
+        return this.state.hasAcceptedTerms
+
+            // These are taken from BankCountry.js in Web-Secure
+            && CONST.BANK_ACCOUNT.REGEX.IBAN.test(this.state.accountNumber.trim())
+            && CONST.BANK_ACCOUNT.REGEX.SWIFT_BIC.test(this.state.routingNumber.trim());
+    }
+
+    addManualAccount() {
+        if (!this.state.hasAcceptedTerms) {
+            console.error('Must accept terms before proceeding');
```

Comment:
Yeah I think it's not really possible since the form technically can't be submitted without accepting the terms. Probably should just remove this part, but was thinking it was more for the developer to see.